src/App.tsx

```diff
@@ -1,24 +1,73 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { TodoList } from './components/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10387;
+
+export enum Error {
+  LOAD = 'load',
+}
+
+export enum FilterBy {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [errorMessage, setErrorMessage] = useState<Error | null>(null);
+
+  const visibleTodos = todos.filter((todo) => {
+    switch (filterBy) {
+      case FilterBy.Active:
+        return !todo.completed;
+
+      case FilterBy.Completed:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  const addError = (error: Error) => {
+    setErrorMessage(error);
+    window.setTimeout(() => {
+      setErrorMessage(null);
+    }, 3000);
+  };
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      try {
+        setErrorMessage(null);
+        const todosFromServer = await getTodos(USER_ID);
+
+        setTodos(todosFromServer);
+      } catch (error) {
+        addError(Error.LOAD);
+      }
+    };
+
+    loadTodos();
+  }, []);
+
+  console.log(todos);
```

Comment:
Remove the console.log statement to avoid displaying todos data in the console during runtime.